test/specs/sorting.test.js

```diff
@@ -0,0 +1,38 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { pages } = require('../pages/Pages');
+
+describe('My First Test', () => {
+  beforeEach(async () => {
+    await pages.loginPage.navigate();
+    await pages.loginPage.performLogin('standard_user', 'secret_sauce');
+    await expect(pages.inventoryPage.headerTitle).toBeExisting();
+    await expect(pages.inventoryPage.headerTitle).toBeExisting();
+  });
+  it('Verify items sorting by Name (A to Z)', async () => {
+    const sortedAZBefore = await pages.inventoryPage.sortedItemsNames('asc');
```

Comment:
let`s use expectedNamesOrder

Reply:
are we taking elements from the page as it is? or are we doing smth with them before?